src/dune_rules/coq_lib.ml

```diff
@@ -237,19 +240,29 @@ module DB = struct
                   | Some _ -> Resolve.Memo.return ()
                   | None -> Error.private_deps_not_allowed ~loc theory_name
               in
-
               (loc, theory))
         in
+        let theories =
+          Resolve.O.(
+            let* boot = boot in
+            match boot with
+            | Some boot when use_stdlib ->
+              let+ theories = theories in
+              (* TODO: if lib is boot, don't add boot dep *)
+              (* maybe use the loc for boot? *)
+              (Loc.none, boot) :: theories
+            | Some _ | None -> theories)
```

Comment:
```suggestion
          let open Resolve.O in
          let* boot = boot in
          match boot with
          | Some boot when use_stdlib ->
            let+ theories = theories in
            (* TODO: if lib is boot, don't add boot dep *)
            (* maybe use the loc for boot? *)
            (Loc.none, boot) :: theories
          | Some _ | None -> theories
```

Reply:
Also should we try and address this comment?

Reply:
Ok, done.